PayoutCurve.md

```diff
@@ -158,9 +157,8 @@ where a spline is made up of polynomial pieces so that the resulting interpolati
 
 #### Polynomial Serialization
 
-1. type: 42792 (`polynomial_payout_curve_piece`)
-2. data:
-   * [`u16`:`num_pts`]
+1. data:
```

Comment:
Shouldn't this be a sub type? And then you can easily parse the difference between `Polynomial`/`Hyperbola` curve pieces easily?

Reply:
Yes you're right, fixed.